### DISCUSSION TITLE
Make v-navigation-drawer open from top to bottom (dropdown)

### DISCUSSION THREAD

**Discussion by saharpe** [vuetifyjs/vuetify#13448] [Q&A]
Hey, im trying to make a v-navigation-drawer component to open up from top to bottom, like a dropdown.
I couldn't find anything about it anywhere, in the API documentation there are only "bottom","right","left" properties..
i would like to make the drawer open from the top.

what i am trying to achieve is very similar to that:
https://codepen.io/ngochuynh/pen/yKrvVZ

Any advice is more than welcome

**Comment by manuvandervorst** [comment]
The v-app-bar is by default at the top of the application. The height value can be changed dynamically if you would want the rest of the application to lower when opening the menu. If you would like the rest of the application to stay the same when the menu is opened I would recommend  using a v-card. :))
